docs/technical/installation_controller.md

```diff
@@ -0,0 +1,38 @@
+# Installation Controller
+
+The reconciliation flow of the installation controller is described in detail [here](./reconciliation_flow.md).
+It executes one of the following main operations:
+
+- a deletion,
+- a [standard reconcile](#standard-reconcile),
+- a [force reconcile](#force-reconcile),
+- or a [re-check of completed installation](#re-check-completed-installation).
```

Comment:
first condition: phase == combined phase of sub-objects.
Does it mean all sub-objects must have the same phase or shall they all be successful? If latter, pls fix.

Reply:
I added a paragraph about the phases and what the "combined phase" is.